shared/bash_remediation_functions/perform_audit_rules_privileged_commands_remediation.sh

```diff
@@ -103,13 +103,12 @@ do
 		base_search=$(sed -e '/-a always,exit/!d' -e '/-F path='"${sbinary_esc}"'[^[:graph:]]/!d'		\
 				-e '/-F path=[^[:space:]]\+/!d'						\
 				-e '/-F auid>='"${min_auid}"'/!d' -e '/-F auid!=\(4294967295\|unset\)/!d'	\
+				-e '/-F perm=[rwxa]\+/d' \
```

Comment:
@vojtapolasek Could you clarify what issue this is fixing?

Reply:
The `base_search` still needs to look for audit rules setting up a specific `-F perm=`.
But then, instead of removing the code block below (former lines 130 to 154) the block should be changed to remove the `perm` filter.

Test scenario `augenrules_rules_with_perm_x.fail.sh`  scenario is passing, but in the end it duplicates the audit rules:
```
-a always,exit -F path=/usr/bin/passwd -F perm=x -F auid>=1000 -F auid!=unset -k privileged
...
-a always,exit -F path=/usr/bin/passwd -F auid>=1000 -F auid!=unset -F key=privileged
```